axonserver/src/test/java/io/axoniq/axonserver/taskscheduler/LocalTaskManagerTest.java

```diff
@@ -0,0 +1,190 @@
+                                                   completableFuture.completeExceptionally(new RuntimeException(
+                                                           "Failed, don't try again"));
+                                               } else {
+                                                   completableFuture.complete(null);
+                                               }
+                                               return completableFuture;
+                                           },
+                                           repository,
+                                           new JacksonTaskPayloadSerializer(),
+                                           new PlatformTransactionManager() {
+                                               @Override
+                                               public TransactionStatus getTransaction(TransactionDefinition definition)
+                                                       throws TransactionException {
+                                                   return new SimpleTransactionStatus();
+                                               }
+
+                                               @Override
+                                               public void commit(TransactionStatus status)
+                                                       throws TransactionException {
+
+                                               }
+
+                                               @Override
+                                               public void rollback(TransactionStatus status)
+                                                       throws TransactionException {
+
+                                               }
+                                           },
+                                           scheduler,
+                                           scheduler.clock());
+        testSubject.start();
+    }
+
+    @Test
+    public void createLocalTask() {
+        testSubject.createLocalTask("DummyHandler",
+                                    new UserInfo("user", Collections.emptySet()),
+                                    Duration.ofSeconds(5));
+        assertEquals(1, tasks.size());
+        scheduler.timeElapses(5, TimeUnit.SECONDS);
+        assertEquals(0, tasks.size());
+    }
+
+    @Test
+    public void createLocalTaskAfter11Minutes() {
+        testSubject.createLocalTask("DummyHandler",
+                                    new UserInfo("user", Collections.emptySet()),
+                                    Duration.ofMinutes(11));
+        assertEquals(1, tasks.size());
+        assertEquals(1, scheduler.tasks());
+        scheduler.timeElapses(5, TimeUnit.MINUTES);
+        assertEquals(1, tasks.size());
+        assertEquals(2, scheduler.tasks());
+
+        scheduler.timeElapses(6, TimeUnit.MINUTES);
+        assertEquals(0, tasks.size());
+        assertEquals(1, scheduler.tasks());
```

Comment:
Slightly at a lose here. The remaining task on the "Scheduler" is the task to retrieve potential tasks, right?